pages/linux/dnf5.md

```diff
@@ -0,0 +1,37 @@
+# dnf5
+
+> Package management utility for RHEL, Fedora, and CentOS (it replaces dnf, which in turn replaced yum).
```

Comment:
Since Linux is case sensitive "dnf" is not the same as "DNF" nor is "YUM" the same as "yum".

Reply:
> Since Linux is case sensitive "dnf" is not the same as "DNF" nor is "YUM" the same as "yum".

Just now went through the other pages using the terms, seems like you are right we seem to use the case-sensitive `dnf` and `yum`. I will mark this conversation as resolved.